kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/LoadBalancerClientTest.java

```diff
@@ -484,19 +465,22 @@ public void run() {
                         .withHeader("Content-Type", "application/xml")
                         .withBody("<response type=\"SUCCESS\" msg=\"Kie Server info\">\n" +
                                 "  <kie-server-info>\n" +
-                                "    <version>background scan</version>\n" +
+                                "    <server-id>background scan</server-id>\n" +
                                 "  </kie-server-info>\n" +
                                 "</response>"))
                 .willSetStateTo("After success 2"));
         logger.debug(" Current wireMockServer1 stub count =" + wireMockServer1.listAllStubMappings().getMappings().size());
 
         // Make call to failing request
+
         try {
             response = client.getServerState();
             fail("Unexpected successful request");
         } catch (KieServerHttpRequestException e) {
-            // expect failure
+            logger.debug("Expected failure Endpoint", e);
         }
+        ((AbstractKieServicesClientImpl) client).getLoadBalancer().setCheckFailedEndpoint(true);
+        ((AbstractKieServicesClientImpl) client).getLoadBalancer().checkFailedEndpoints().get(5, TimeUnit.SECONDS);
```

Comment:
Scenarios after enabling checkFailedEnpoints are not taking into account any failure endpoint. I think we should enable it before making the calls to the kie-server